guide/contribute/propose.md

```diff
@@ -0,0 +1,173 @@
+
+{% include youtube.html id="nt02IS6oIJw" %}
+
+1. [Create a fork (clone)](#forking-design-guide-repository)
+2. [Create a branch](#creating-a-branch)
+3. [Manage files](#managing-files)
+4. [Apply changes](#applying-changes)
+5. [Create a pull request](#creating-a-pull-request)
+6. [Applying additional changes](#applying-additional-changes)
+
+#### Forking Design Guide repository
+
+The first step is to fork the Bitcoin Design Guide repository. Forking a repository means that you're creating an exact clone (copy) of the Bitcoin Design Guide locally on your computer and in a cloud on your GitHub.com account.
+
+To fork a repository[^1]:
+
+- Sign in to GitHub desktop
+- File > `Clone repository`
+- Switch to the URL tab
+- Paste `https://github.com/BitcoinDesign/Guide/` into the URL field.
+- Click `Clone`
+
+{% include picture.html
+   image = "/assets/images/guide/contribute/propose/clone-repository.png"
+   retina = "/assets/images/guide/contribute/propose/clone-repository@2x.png"
+   alt-text = "Cloning a repository"
+   width = 400
+   height = 400
+%}
+
+#### Creating a branch
+
+A branch[^2] allows you to have a _contained area_ where you can make any changes, without going out of sync with the cloned version.
+
+The cloned version will reside on the `master branch`, while any changes should be done on separate branches. This isn't a strict rule, it's the best practice that would allow you to easier sync your copy with the original (upstream) version of the design guide.
+
+
+{% include picture.html
+   image = "/assets/images/guide/contribute/propose/create-branch.png"
+   retina = "/assets/images/guide/contribute/propose/create-branch@2x.png"
+   alt-text = "Creating a branch"
+   width = 800
+   height = 542
+%}
+
+#### Managing files
+
+When a new branch is created, while making sure it's selected, click on either `Open in Text Editor` or`Show in explorer` to browse through the folders and files. You can use a search (find) function `Ctrl+F/ cmd+F` to search for a specific keyword that can help identify the file you would like to edit.
+
+#### Applying changes
+
+After identifying a file or folder where you want to make a change, it's time to make your changes to the text, code, or visuals. Whenever you're finished, make sure to `Save` the file (File > Save) in a _text editor_ or use a shortcut `Ctrl+S/cmd+S`.
+
+Please make sure that your changes are consistent with our [content](content-guidelines), [illustration](illustration-guidelines) and [formatting](formatting) guidelines.
+
+{% include youtube.html id="TjydTyOK4Wo" %}
+
+##### Committing
+
+Saving changes in a text editor automatically triggers the GitHub Desktop application to showcase them. You can review the changes quickly, and then make a commit[^3].
+
+Commits should contain _meaningful changes_ grouped together.
+
+**Do**
+- Add good textual context to your commit
+
+**Don't**
+- Abuse the commit function to gain commits, not every change should be a commit.
```

Comment:
What is gaining commits? Does it refer to somebody trying to beef up their Github stats?

Reply:
Correct, should I remove it, as it may be irrelevant? Commits can always be squashed.

Reply:
Probably too advanced for an introductory page. I'd remove it.

Reply:
9e65bf092d0be6b5c5ecafc0f015c17407fcb7aa